sql/core/src/main/scala/org/apache/spark/sql/package.scala

```diff
@@ -44,4 +44,13 @@ package object sql {
   type Strategy = SparkStrategy
 
   type DataFrame = Dataset[Row]
+
+  /**
+   * Metadata key which is used to write Spark version in the followings:
+   * - Parquet file metadata
+   * - ORC file metadata
+   *
+   * Note that Hive table property `spark.sql.create.version` also has Spark version.
+   */
+  private[sql] val CREATE_VERSION = "org.apache.spark.sql.create.version"
```

Comment:
Is this a pre-existing key? Seems that `org.apache.spark.version` should be enough.